web_external/views/map/MapPanel.js

```diff
@@ -418,6 +430,20 @@ const MapPanel = Panel.extend({
             trigger: 'hover'
         };
         this.$('.m-save-current-baselayer').tooltip(tooltipProperties);
+
+        var screenshotButtonContainer = $(this.screenshotWidget.canvas());
+        screenshotButtonContainer.empty().append(screenshotButtonTemplate());
+        screenshotButtonContainer.find('button').on('click', () => {
+            var layers = this.map.layers().filter((layer) => layer.layerName !== 'ui');
+            this.map.screenshot(layers, { wait: 'idle' }).then((image) => {
```

Comment:
The `map.screenshot` function should either be `map.screenshot(layers, undefined, undefined, {wait: 'idle'})` or `map.screenshot({layers: layers, wait: 'idle'})`.